build/webpack.config.content.prod.js

```diff
@@ -0,0 +1,24 @@
+// Any copyright is dedicated to the Public Domain.
+// http://creativecommons.org/publicdomain/zero/1.0/
+
+import webpack from 'webpack';
+import contentConfig from './webpack.config.default.content';
+
+export default {
+  ...contentConfig,
+  plugins: [
+    ...contentConfig.plugins,
+    new webpack.NoErrorsPlugin(),
+    new webpack.DefinePlugin({
+      'process.env': {
+        NODE_ENV: JSON.stringify('production'),
+      },
+    }),
+  ],
+  debug: true,
```

Comment:
This is a flag that makes webpack and (more importantly) plugins spew stuff to stdout when building when "bad things might be happening". I found a template webpackrc file that had this, so I turned it on too. But since we're not really sure we need this, I'll remove it altogether.